core/domain/base_model_validators_test.py

```diff
@@ -233,50 +233,71 @@ def test_external_model_fetcher_with_user_settings_raise_error(self):
             base_model_validators.ExternalModelFetcherDetails(
                 'committer_ids', user_models.UserSettingsModel,
                 [
-                    feconf.MIGRATION_BOT_USER_ID, 'User-1',
+                    feconf.MIGRATION_BOT_USER_ID, 'uid_%s' % ('a' * 32),
                     self.PSEUDONYMOUS_ID
                 ]
             )
 
+    def test_external_model_fetcher_with_invalid_id(self):
+        with self.assertRaisesRegexp(
+            Exception,
+            'A model id in the field \'mock_field\' '
+            'is empty'):
+            base_model_validators.ExternalModelFetcherDetails(
+                'mock_field', MockModel, ['']
+            )
+
+    def test_user_setting_model_fetcher_with_invalid_id(self):
+        with self.assertRaisesRegexp(
+            Exception,
+            'The user id User-1 in the field \'mock_field\' '
+            'is invalid'):
+            base_model_validators.UserSettingsModelFetcherDetails(
+                'mock_field', ['User-1'],
+                may_contain_system_ids=False,
+                may_contain_pseudonymous_ids=False
+            )
+
     def test_may_contain_system_users_filters_system_ids(self):
         user_settings_model = (
             base_model_validators.UserSettingsModelFetcherDetails(
                 'committer_ids',
-                [feconf.MIGRATION_BOT_USER_ID, 'User-1'],
+                [feconf.MIGRATION_BOT_USER_ID, 'uid_%s' % ('a' * 32)],
```

Comment:
Maybe define the userID in the setUp function

Reply:
Done.